tests/acceptance/features/apiAntivirus/antivirusMain.feature

```diff
@@ -0,0 +1,101 @@
+		And user "user0" uploads chunk file "3" of "3" with "CCCCC" to "/myChunkedFile.txt" using the API
+		Then as "user0" the file "/myChunkedFile.txt" should exist
+		And the content of file "/myChunkedFile.txt" for user "user0" should be "AAAAABBBBBCCCCC"
+
+	Scenario: A small file with a virus cannot be uploaded in chunks using API version 1 and old DAV path
+		Given using API version "1"
+		And using old DAV path
+		When user "user0" uploads chunk file "1" of "3" with "This kitten " to "/myChunkedFile.txt" using the API
+		And user "user0" uploads chunk file "2" of "3" with "is a nasty " to "/myChunkedFile.txt" using the API
+		And user "user0" uploads chunk file "3" of "3" with "virus" to "/myChunkedFile.txt" using the API
+		Then the last lines of the log file should contain log-entries containing these attributes:
+			| user  | app             | method | message               |
+			| user0 | files_antivirus | PUT    | Infected file deleted |
+		And as "user0" the file "/myChunkedFile.txt" should not exist
+
+	Scenario: A small file without a virus can be uploaded in chunks using API version 1 and new DAV path
+		Given using API version "1"
+		And using new DAV path
+		When user "user0" creates a new chunking upload with id "chunking-42" using the API
+		And user "user0" uploads new chunk file "1" with "AAAAA" to id "chunking-42" using the API
+		And user "user0" uploads new chunk file "2" with "BBBBB" to id "chunking-42" using the API
+		And user "user0" uploads new chunk file "3" with "CCCCC" to id "chunking-42" using the API
+		And user "user0" moves new chunk file with id "chunking-42" to "/myChunkedFile.txt" with size 15 using the API
+		Then as "user0" the file "/myChunkedFile.txt" should exist
+		And the content of file "/myChunkedFile.txt" for user "user0" should be "AAAAABBBBBCCCCC"
+
+	Scenario: A small file with a virus cannot be uploaded in chunks using API version 1 and new DAV path
+		Given using API version "1"
+		And using new DAV path
+		When user "user0" creates a new chunking upload with id "chunking-42" using the API
+		And user "user0" uploads new chunk file "1" with "This kitten " to id "chunking-42" using the API
+		And user "user0" uploads new chunk file "2" with "is a nasty " to id "chunking-42" using the API
+		And user "user0" uploads new chunk file "3" with "virus" to id "chunking-42" using the API
+		And user "user0" moves new chunk file with id "chunking-42" to "/myChunkedFile.txt" with size 28 using the API
+		Then the last lines of the log file should contain log-entries containing these attributes:
+			| user  | app             | method | message               |
+			| user0 | files_antivirus | MOVE   | Infected file deleted |
+		And as "user0" the file "/myChunkedFile.txt" should not exist
+
+	Scenario: A small file without a virus can be uploaded in chunks using API version 2 and new DAV path
+		Given using API version "1"
```

Comment:
sorry to point again - what is the reason for stating `version 2` in the scenario description
> A small file without a virus can be uploaded in chunks using API version 2 and new DAV path

But then using v1 in Given step
> Given using API version "1"

Reply:
Well spotted - I put that cut-paste-forget-to-edit error there just to see if reviewers were reviewing well ;)
Fixed, thanks